src/main/java/org/openstreetmap/atlas/checks/validation/intersections/AtGradeSignPostCheck.java

```diff
@@ -231,9 +231,9 @@ else if (roundAboutInEdgeToOutEdgeMap != null)
         final List<String> identifiers = this.getIdentifiers(entitiesToBeFlagged);
```

Comment:
This list also might need to use OSM IDs. That would need to be ubdated in `getIdentifiers`.